city_scrapers/spiders/chi_committee_on_design.py

```diff
@@ -0,0 +1,154 @@
+from pickle import FALSE, TRUE
+from datetime import datetime
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiCommitteeOnDesignSpider(CityScrapersSpider):
+    name = "chi_committee_on_design"
+    agency = "Comittee On Design"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.chicago.gov/city/en/depts/dcd/supp_info/committee-on-design.html"
+    ]
+
+    def parse(self, response):
+
+        # Parse strong text on page to get a list of all years
+        # each has a table of meetings under it
+        years = response.css("strong::text").getall()
+        years = years[3:]
+        years = [i[0:4] for i in years]
+
+        tablecount = 0
+        for table in response.css("table"):  # loop through every meeting table
+            # holds all meetings for a given year (loop)
+            total = []
+            # list of raw HTML for a given meeting
+            # to later get meeting agenda if link exists
+            total_clean_dates = []
+            for item in table.css("td"):
+                dates = item.css("p::text").getall()
+                # taking raw HTML and splitting by date
+                # to later pull meeting agenda link
+                clean_dates = item.css("p").get().split("<br>")
+                total_clean_dates += clean_dates
+                # loop through text from the table, removing non-date strings
+                for i in range(len(dates)):
+                    curr = dates[i]
+                    d = curr.strip()
+                    d = d.rstrip("\xa0")
+                    dates[i] = d
+                for d in dates:
+                    if d[0] == "•":
+                        dates.remove(d)
+                total += dates  # add dates to list of meetings
+            print("SIZE OF TOTAL " + str(total))
+            print("SIZE OF CLEAN DATES " + str(clean_dates))
+            # get current year based on which table we are on
+            curyear = years[tablecount]
+            for i in range(len(total)):
+                meet = total[i]  # current meeting we are working with
+                canceled = FALSE
+                splitMeet = meet.split()
+                # meeting string is in format "month day" or "month day canceled"
+                if len(splitMeet) > 2:
+                    if splitMeet[2].strip() == "Canceled":
+                        canceled = TRUE
+                        meet = splitMeet[0] + " " + splitMeet[1]
+                    else:  # if not "canceled", it is unexpected so we throw an error
+                        raise ValueError("Unknown addition to date time")
+                clean_date = total_clean_dates[i]
+                clean_date_list = clean_date.split('"')
+                agendaLink = ""
```

Comment:
Python variables should follow `snake_case` naming convention

Reply:
Done!